src/index.js

```diff
@@ -49,10 +48,10 @@ class WebSockets {
     }
 
     return multiaddrs.filter((ma) => {
-      if (includes(ma.protoNames(), 'ipfs')) {
-        ma = ma.decapsulate('ipfs')
-      }
-      return mafmt.WebSockets.matches(ma) || mafmt.WebSocketsSecure.matches(ma)
+      return mafmt.WebSockets.matches(ma) ||
+        mafmt.WebSocketsSecure.matches(ma) ||
+        mafmt.WebSocketsIPFS.matches(ma) ||
+        mafmt.WebSocketsSecureIPFS.matches(ma)
```

Comment:
Following https://github.com/whyrusleeping/js-mafmt/pull/16#discussion_r133625774, if you do multiple assertions anyways, why having some mix and some not. Just assert for IPFS separately. 

From this Boolean assertion, seems that having IPFS is actually not relevant.